### DISCUSSION TITLE
Start Menu not working, no change to Task Bar

### DISCUSSION THREAD

**Discussion by Theguffy1990** [valinet/ExplorerPatcher#801] [General]
Thank you for taking the time to do this.

Hello,

I apologies if I am not following a convention, but I will try to use the "may be used as a template?" post from a while back. 

Windows Version: Windows 11 Enterprise 21996.1
Fresh install/upgrade: Upgrade (from fresh Win10 install)
EP Version: 22000.469.41.16 (Release), 22000.469.42.5 (latest PR), 22000.469.42.3 (2nd latest PR), 22000.469.42.2 (3rd latest PR)
StartIsBack applied: False
Any other plugins/"fixes" applied: False

As briefly mentioned in #31 (though I feel as though it's not relevant anymore), my Start Menu becomes completely unresponsive after what I assume is a failed install. I downloaded the latest release from 11 days ago, and once it finished its rather fast download, double clicked on it and granted it elevated permissions. My pinned Icons disappear and the downloads folder closed (from what I assume was a restart). After around 20-30 seconds, my pinned Icons reappear, but do not seem changed. I click the Start Menu icon and the cursor briefly flickers the loading sprite then goes back to normal, but nothing more happens. I try again after a few seconds as I assumed I had missed somehow, but same behavior. I pressed the "Windows" key on my keyboard, and the same behavior again. I leave it for a moment to get a drink assuming that it is still installing, but after 5 minutes, nothing has appeared, and the same behavior is observed. 

After restarting my computer (hard reset as I didn't have the power button obviously), I still do not have access to the Start Menu, and the Taskbar still looks the same. This is when I tried the latest PR, but it made no change, as with 2nd and 3rd. I sloppily did not even think to uninstall the previous version between new releases, but when I went to the Win10-esque Programs & Features list, it didn't show. I went to the Win7-esque logical list, but it also wasn't there. I came to the site to find out if there was another way, and found the rename method (thank you for this) worked. This opened with a dialogue box and everything. I installed latest, but had the same issue. 

If there's a log or something for this, I'll happy supply it. I couldn't see any other issue like this that had any kind of detail (shoutout to that screenshot of the taskbar guy).

Thank you.

**Comment by valinet** [comment]
Moved to Discussions as per https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems

**Comment by Poopooracoocoo** [comment]
Strange Windows build :/ You should be on build 22000.469 I think. You could start troubleshooting by updating to the latest release channel build.

**Reply by Theguffy1990** [reply]
See, I did think I was out of sync with the update channels, but no matter what I do, it does not want to update. I've tried fresh installs (from scratch), this "upgrade" version, and even joining the Insider Program to "force" an update. Nothing happens, and obviously I'll only get an Insider Program update when they release one. As the system is only around 30 hours old, I doubt it will see one at all.

**Reply by valinet** [reply]
Install from the ISO image from here: https://www.microsoft.com/software-download/windows11

That will get you the latest public build based on 22000, which at the current time is 22000.434, I think. 21996 is the leaked version from the summer, it’s very old and EP was never tested on that.

**Reply by Theguffy1990** [reply]
Ah, it would appear I am incredibly dim. Yup, the ISO I had was probably downloaded around then. I did look to see if I could find a more current version, but clearly I didn't look hard enough. I downloaded the ISO, rufus'd a pen drive, clean installed then reactivated my Windows (it got angry because I did a processor/MoBo swap). I'm up and running, and have a functional taskbar again. 

Thank you so much for your help, and your work! :)

**Reply by valinet** [reply]
You’re welcome, no worries, I am glad that you sorted it out!
